### PR TITLE
feat: adapt the release-candidate PR message, and fallback when rebasing

### PR DESCRIPTION
**Pull request by david-paulus-saad**

### CODE REVIEW

**Comment by david-paulus-saad** [comment]
Welcome to the release candidate!

### Done Issues in Planner and Orders (12 %)
- [ ] [[Rabbitmq] A queue has unroutable messages](https://linear.app/impargo/issue/POS-139/[rabbitmq]-a-queue-has-unroutable-messages)
- [ ] [[Planner] charged distance without a tolls](https://linear.app/impargo/issue/POS-138/[planner]-charged-distance-without-a-tolls)
- [ ] [[Backend] kama-import job usually fails due to network errors](https://linear.app/impargo/issue/POS-137/[backend]-kama-import-job-usually-fails-due-to-network-errors)
- [ ] [Add more extensive routing test cases](https://linear.app/impargo/issue/POS-136/add-more-extensive-routing-test-cases)
- [ ] [Route calculation fails with BAD_USER_INPUT](https://linear.app/impargo/issue/POS-135/route-calculation-fails-with-bad-user-input)
- [ ] [[Feature] Add email to the contact details of an address in CA](https://linear.app/impargo/issue/POS-134/[feature]-add-email-to-the-contact-details-of-an-address-in-ca)
- [ ] [[Feature] Loading unloading units in the CA](https://linear.app/impargo/issue/POS-133/[feature]-loading-unloading-units-in-the-ca)
- [ ] [[Feature] Send notification email when uploading a document in SP](https://linear.app/impargo/issue/POS-132/[feature]-send-notification-email-when-uploading-a-document-in-sp)
- [ ] [[Research] Re-architect orders to improve query performance](https://linear.app/impargo/issue/POS-131/[research]-re-architect-orders-to-improve-query-performance)
- [ ] [[CargoApps] Update order import](https://linear.app/impargo/issue/POS-130/[cargoapps]-update-order-import)
- [ ] [[Guide] Update import orders guide](https://linear.app/impargo/issue/POS-129/[guide]-update-import-orders-guide)
- [ ] [[Backend] Expose improved importing orders API](https://linear.app/impargo/issue/POS-128/[backend]-expose-improved-importing-orders-api)
- [ ] [[Planner] Crash when using mapbox as map provider](https://linear.app/impargo/issue/POS-126/[planner]-crash-when-using-mapbox-as-map-provider)
- [ ] [[CargoApps] Route calculation inconsistent between prod / dev](https://linear.app/impargo/issue/POS-125/[cargoapps]-route-calculation-inconsistent-between-prod-dev)
- [ ] [[Orders] Sharing disabled to driver for assigned orders](https://linear.app/impargo/issue/POS-123/[orders]-sharing-disabled-to-driver-for-assigned-orders)
- [ ] [[Backend] Order import and mass tolls doesn't use route-calculation service](https://linear.app/impargo/issue/POS-127/[backend]-order-import-and-mass-tolls-doesnt-use-route-calculation)
- [ ] [[CargoApps] Order assignment via link doesn't work after status change](https://linear.app/impargo/issue/POS-122/[cargoapps]-order-assignment-via-link-doesnt-work-after-status-change)
- [ ] [[Orders] My orders and incoming orders checkbox not working](https://linear.app/impargo/issue/POS-121/[orders]-my-orders-and-incoming-orders-checkbox-not-working)
- [ ] [[Planner] Saving updates to order fails](https://linear.app/impargo/issue/POS-120/[planner]-saving-updates-to-order-fails)
- [ ] [[Planner] Re-opening same order doesn't work](https://linear.app/impargo/issue/POS-119/[planner]-re-opening-same-order-doesnt-work)
- [ ] [[CargoApps] Adjusted price per km ignored when adding order](https://linear.app/impargo/issue/POS-118/[cargoapps]-adjusted-price-per-km-ignored-when-adding-order)
- [ ] [[CargoApps] Login fails without error message](https://linear.app/impargo/issue/POS-117/[cargoapps]-login-fails-without-error-message)
- [ ] [[Planner/AddOrder] Additional stop data adaptions](https://linear.app/impargo/issue/POS-116/[planneraddorder]-additional-stop-data-adaptions)
- [ ] [[Orders] Additional stop based tracking data in order drawer](https://linear.app/impargo/issue/POS-115/[orders]-additional-stop-based-tracking-data-in-order-drawer)
- [ ] [[Planner] Wrong price after editing vehicles or drivers](https://linear.app/impargo/issue/POS-114/[planner]-wrong-price-after-editing-vehicles-or-drivers)
- [ ] [[Mixpanel] Track user journey of a new user from Log in to Drop off](https://linear.app/impargo/issue/POS-113/[mixpanel]-track-user-journey-of-a-new-user-from-log-in-to-drop-off)
- [ ] [Sorting query memory limit backend](https://linear.app/impargo/issue/POS-112/sorting-query-memory-limit-backend)
- [ ] [[Backend] Sorting functionality of orders by dates is broken](https://linear.app/impargo/issue/POS-111/[backend]-sorting-functionality-of-orders-by-dates-is-broken)
- [ ] [[feature] Add email to the contact details of an address](https://linear.app/impargo/issue/POS-110/[feature]-add-email-to-the-contact-details-of-an-address)
- [ ] [[feature] Add email to the contact details of an address](https://linear.app/impargo/issue/POS-109/[feature]-add-email-to-the-contact-details-of-an-address)
- [ ] [[feature] Add loading meters to ShipperPortal](https://linear.app/impargo/issue/POS-108/[feature]-add-loading-meters-to-shipperportal)
- [ ] [[feature] Allow uploading document to already assigned orders](https://linear.app/impargo/issue/POS-107/[feature]-allow-uploading-document-to-already-assigned-orders)
- [ ] [[feature] Allow uploading document to already assigned orders](https://linear.app/impargo/issue/POS-106/[feature]-allow-uploading-document-to-already-assigned-orders)
- [ ] [[feature] Loading / unloading reference for loading units](https://linear.app/impargo/issue/POS-105/[feature]-loading-unloading-reference-for-loading-units)
- [ ] [[feature] Loading / unloading reference](https://linear.app/impargo/issue/POS-104/[feature]-loading-unloading-reference)
- [ ] [[feature] ShipperPortal transport filters](https://linear.app/impargo/issue/POS-103/[feature]-shipperportal-transport-filters)
- [ ] [[feature] ShipperPortal transport filters](https://linear.app/impargo/issue/POS-102/[feature]-shipperportal-transport-filters)
- [ ] [[Integrations] Cleanup billing integration after full launch](https://linear.app/impargo/issue/POS-101/[integrations]-cleanup-billing-integration-after-full-launch)
- [ ] [Sync notes and activities](https://linear.app/impargo/issue/POS-100/sync-notes-and-activities)
- [ ] [Sync basic company and person data](https://linear.app/impargo/issue/POS-99/sync-basic-company-and-person-data)
- [ ] [[Maintenance] Research react 18 upgrade possibility](https://linear.app/impargo/issue/POS-98/[maintenance]-research-react-18-upgrade-possibility)
- [ ] [Stop-based Tracking KPI - CargoApps](https://linear.app/impargo/issue/POS-95/stop-based-tracking-kpi-cargoapps)
- [ ] [[CargoApps] Fire notification event after accepting a takeover](https://linear.app/impargo/issue/POS-94/[cargoapps]-fire-notification-event-after-accepting-a-takeover)
- [ ] [[Landing Page] Investigate SEO issues in truck bans blog article](https://linear.app/impargo/issue/POS-93/[landing-page]-investigate-seo-issues-in-truck-bans-blog-article)
- [ ] [[Research] Offer Management Process](https://linear.app/impargo/issue/POS-92/[research]-offer-management-process)
- [ ] [Offer Details View](https://linear.app/impargo/issue/POS-91/offer-details-view)
- [ ] [Create Order from Offer](https://linear.app/impargo/issue/POS-90/create-order-from-offer)
- [ ] [Reorganize Orders Page](https://linear.app/impargo/issue/POS-89/reorganize-orders-page)
- [ ] [My Offer Tab](https://linear.app/impargo/issue/POS-88/my-offer-tab)
- [ ] [Offer Link Page - Accept / Decline](https://linear.app/impargo/issue/POS-87/offer-link-page-accept-decline)

### Done Issues in Design (9 %)
- [ ] [Step 1: Set up project structure.](https://linear.app/impargo/issue/DES-91/step-1-set-up-project-structure)
- [ ] [Chore: Update stop-based tracking with additional stop data](https://linear.app/impargo/issue/DES-90/chore-update-stop-based-tracking-with-additional-stop-data)
- [ ] [DriverApp - Chat screen](https://linear.app/impargo/issue/DES-89/driverapp-chat-screen)
- [ ] [DriverApp - Notifications](https://linear.app/impargo/issue/DES-88/driverapp-notifications)
- [ ] [DriverApp - Settings screen](https://linear.app/impargo/issue/DES-87/driverapp-settings-screen)
- [ ] [Offer management - adaptations](https://linear.app/impargo/issue/DES-86/offer-management-adaptations)
- [ ] [[feature/change] Geo-Fencing for Stop based tracking](https://linear.app/impargo/issue/DES-85/[featurechange]-geo-fencing-for-stop-based-tracking)
- [ ] [Notification per Stop](https://linear.app/impargo/issue/DES-84/notification-per-stop)
- [ ] [[feature] ShipperPortal transport filters](https://linear.app/impargo/issue/DES-83/[feature]-shipperportal-transport-filters)
- [ ] [[feature] Loading / unloading reference for loading units](https://linear.app/impargo/issue/DES-82/[feature]-loading-unloading-reference-for-loading-units)
- [ ] [[feature] Allow uploading document to already assigned orders](https://linear.app/impargo/issue/DES-81/[feature]-allow-uploading-document-to-already-assigned-orders)
- [ ] [[feature] Add loading meters to ShipperPortal](https://linear.app/impargo/issue/DES-80/[feature]-add-loading-meters-to-shipperportal)
- [ ] [[feature] Add email to the contact details of an address](https://linear.app/impargo/issue/DES-79/[feature]-add-email-to-the-contact-details-of-an-address)
- [ ] [Subscriptions: improve displaying cancelled subscriptions](https://linear.app/impargo/issue/DES-78/subscriptions-improve-displaying-cancelled-subscriptions)
- [ ] [Chore: Create high level PRD document for different flows.](https://linear.app/impargo/issue/DES-77/chore-create-high-level-prd-document-for-different-flows)
- [ ] [[feature] [planner] show which stop cannot be reached and why](https://linear.app/impargo/issue/DES-76/[feature]-[planner]-show-which-stop-cannot-be-reached-and-why)
- [ ] [Squad Builder: Product Design](https://linear.app/impargo/issue/DES-75/squad-builder-product-design)
- [ ] [[PDF] Include leg distances in exported PDFs](https://linear.app/impargo/issue/DES-74/[pdf]-include-leg-distances-in-exported-pdfs)
- [ ] [Chore: Design System Components](https://linear.app/impargo/issue/DES-73/chore-design-system-components)
- [ ] [Add order modal stops card summary](https://linear.app/impargo/issue/DES-72/add-order-modal-stops-card-summary)
- [ ] [Add order modal, stops title inconsistent](https://linear.app/impargo/issue/DES-71/add-order-modal-stops-title-inconsistent)
- [ ] [[Design] Standards for transitions and animation on DriverApp](https://linear.app/impargo/issue/DES-70/[design]-standards-for-transitions-and-animation-on-driverapp)
- [ ] [DriverApp - allow swiping on Home bottom sheet to change orders](https://linear.app/impargo/issue/DES-69/driverapp-allow-swiping-on-home-bottom-sheet-to-change-orders)
- [ ] [Research best practices for Drawer on the Driver App](https://linear.app/impargo/issue/DES-68/research-best-practices-for-drawer-on-the-driver-app)
- [ ] [Step 1: LO-Fi: additional stop data](https://linear.app/impargo/issue/DES-51/step-1-lo-fi-additional-stop-data)
- [ ] [Additional stop details, DriverApp, CargoApps.](https://linear.app/impargo/issue/DES-49/additional-stop-details-driverapp-cargoapps)
- [ ] [[Design] CargoApps disable sending orders without routes to driver](https://linear.app/impargo/issue/DES-5/[design]-cargoapps-disable-sending-orders-without-routes-to-driver)
- [ ] [[Design] DriverApp map zooming / panning behavior](https://linear.app/impargo/issue/DES-6/[design]-driverapp-map-zooming-panning-behavior)
- [ ] [[Design] Create new components for marketing](https://linear.app/impargo/issue/DES-11/[design]-create-new-components-for-marketing)
- [ ] [[Design] Rebranding](https://linear.app/impargo/issue/DES-4/[design]-rebranding)
- [ ] [[Design] Create first illustrations components](https://linear.app/impargo/issue/DES-12/[design]-create-first-illustrations-components)
- [ ] [Inform user to reload the app](https://linear.app/impargo/issue/DES-8/inform-user-to-reload-the-app)
- [ ] [[CargoApps] Enhance the UX of the costprofiles selector.](https://linear.app/impargo/issue/DES-38/[cargoapps]-enhance-the-ux-of-the-costprofiles-selector)
- [ ] [[Eurotransline] Implement feedback to existing designs.](https://linear.app/impargo/issue/DES-56/[eurotransline]-implement-feedback-to-existing-designs)
- [ ] [[Design] Create Corporate Identity document](https://linear.app/impargo/issue/DES-55/[design]-create-corporate-identity-document)
- [ ] [[Product Management] Get feedback for stop based tracking](https://linear.app/impargo/issue/DES-58/[product-management]-get-feedback-for-stop-based-tracking)
- [ ] [[ShipperPortal] Stop-based tracking - ShipperPortal](https://linear.app/impargo/issue/DES-42/[shipperportal]-stop-based-tracking-shipperportal)
- [ ] [[DriverApp] DriverApp stop-based tracking](https://linear.app/impargo/issue/DES-1/[driverapp]-driverapp-stop-based-tracking)
- [ ] [[CargoApps] Stop-based tracking - Orders](https://linear.app/impargo/issue/DES-13/[cargoapps]-stop-based-tracking-orders)
- [ ] [[CargoApps] Stop-based tracking - Fleet](https://linear.app/impargo/issue/DES-7/[cargoapps]-stop-based-tracking-fleet)
- [ ] [[Design - Chrore] Merge: [CargoApps] Favorites category selection #7451](https://linear.app/impargo/issue/DES-31/[design-chrore]-merge-[cargoapps]-favorites-category-selection-7451)
- [ ] [[CargoApps - ShipperPortal - DriverApp] Competitive analysis.](https://linear.app/impargo/issue/DES-59/[cargoapps-shipperportal-driverapp]-competitive-analysis)
- [ ] [[CA - RC - SP] Replace avatar initials with default user icon.](https://linear.app/impargo/issue/DES-39/[ca-rc-sp]-replace-avatar-initials-with-default-user-icon)
- [ ] [[Design] Create illustrations guidelines.](https://linear.app/impargo/issue/DES-16/[design]-create-illustrations-guidelines)
- [ ] [[Mockups] Eurotransline procurement process](https://linear.app/impargo/issue/DES-17/[mockups]-eurotransline-procurement-process)
- [ ] [[Mockups] Eurotransline order execution monitoring process](https://linear.app/impargo/issue/DES-40/[mockups]-eurotransline-order-execution-monitoring-process)
- [ ] [[Mockups] Eurotransline dispatch process](https://linear.app/impargo/issue/DES-41/[mockups]-eurotransline-dispatch-process)
- [ ] [[Design] Create guidelines for marketing materials](https://linear.app/impargo/issue/DES-19/[design]-create-guidelines-for-marketing-materials)
- [ ] [Update theme in figma and design tokens repo](https://linear.app/impargo/issue/DES-43/update-theme-in-figma-and-design-tokens-repo)
- [ ] [[Design] Revisit booking driver app flows](https://linear.app/impargo/issue/DES-3/[design]-revisit-booking-driver-app-flows)

### Done Issues in DevOps (8 %)
- [ ] [[coveralls] Formulate the coveralls issue and try to reproduce it.](https://linear.app/impargo/issue/DEV-27/[coveralls]-formulate-the-coveralls-issue-and-try-to-reproduce-it)
- [ ] [[Backend] Improve deployment and release workflow to reduce time and handle the failures.](https://linear.app/impargo/issue/DEV-26/[backend]-improve-deployment-and-release-workflow-to-reduce-time-and)
- [ ] [[monitoring] adapt the alerting thresholds for some alerts](https://linear.app/impargo/issue/DEV-25/[monitoring]-adapt-the-alerting-thresholds-for-some-alerts)
- [ ] [[DevOps] adapt the Rabbitmq memory](https://linear.app/impargo/issue/DEV-24/[devops]-adapt-the-rabbitmq-memory)
- [ ] [Assign tester improvements](https://linear.app/impargo/issue/DEV-23/assign-tester-improvements)
- [ ] [[Monitoring] Routing error alerts](https://linear.app/impargo/issue/DEV-22/[monitoring]-routing-error-alerts)
- [ ] [[CI] Add helm lint and manifest validation stage](https://linear.app/impargo/issue/DEV-21/[ci]-add-helm-lint-and-manifest-validation-stage)
- [ ] [Set up a review checklist for the DevOps tasks](https://linear.app/impargo/issue/DEV-20/set-up-a-review-checklist-for-the-devops-tasks)
- [ ] [Figure out a reliable way to test database migrations.](https://linear.app/impargo/issue/DEV-19/figure-out-a-reliable-way-to-test-database-migrations)
- [ ] [[release-workflow] implement the new release workflow](https://linear.app/impargo/issue/DEV-18/[release-workflow]-implement-the-new-release-workflow)
- [ ] [QA/review process proposal](https://linear.app/impargo/issue/DEV-17/qareview-process-proposal)
- [ ] [[cms] enable backup and restore from local sql dump](https://linear.app/impargo/issue/DEV-14/[cms]-enable-backup-and-restore-from-local-sql-dump)
- [ ] [[Maintenance] Disable Dependabot](https://linear.app/impargo/issue/DEV-11/[maintenance]-disable-dependabot)
- [ ] [[Optimization] Comparing the cost of current Azure workload against their cost on AWS.](https://linear.app/impargo/issue/DEV-6/[optimization]-comparing-the-cost-of-current-azure-workload-against)
- [ ] [[Monitoring] Distributed tracing Implementation](https://linear.app/impargo/issue/DEV-5/[monitoring]-distributed-tracing-implementation)
- [ ] [[DevOps] Performance Monitoring proposal](https://linear.app/impargo/issue/DEV-4/[devops]-performance-monitoring-proposal)
- [ ] [[DevOps] try using swapping for the routing node pool, and if possible use cost-efficient virtual machines.](https://linear.app/impargo/issue/DEV-7/[devops]-try-using-swapping-for-the-routing-node-pool-and-if-possible)
- [ ] [[Backend] Share duplicated context creation](https://linear.app/impargo/issue/DEV-15/[backend]-share-duplicated-context-creation)
- [ ] [[Backend] Optimize companyOrders query](https://linear.app/impargo/issue/DEV-16/[backend]-optimize-companyorders-query)

### Done Issues in DriverApp & Fleet Squad (14 %)
- [ ] [tracking_enabled_bloc: _CastError: Null check operator used on a null value](https://linear.app/impargo/issue/DFS-97/tracking-enabled-bloc-casterror-null-check-operator-used-on-a-null)
- [ ] [map_bloc: _CastError: Null check operator used on a null value](https://linear.app/impargo/issue/DFS-96/map-bloc-casterror-null-check-operator-used-on-a-null-value)
- [ ] [Connectivity: _CastError: Null check operator used on a null value](https://linear.app/impargo/issue/DFS-95/connectivity-casterror-null-check-operator-used-on-a-null-value)
- [ ] [DriverApp Settings Screen](https://linear.app/impargo/issue/DFS-94/driverapp-settings-screen)
- [ ] [[Driver App] Stop Button freezes when upload pod canceled](https://linear.app/impargo/issue/DFS-93/[driver-app]-stop-button-freezes-when-upload-pod-canceled)
- [ ] [[AdminPortal] ShipperPortal KPIs incosistent](https://linear.app/impargo/issue/DFS-90/[adminportal]-shipperportal-kpis-incosistent)
- [ ] [[ShipperPortal] Infinite eta polling](https://linear.app/impargo/issue/DFS-91/[shipperportal]-infinite-eta-polling)
- [ ] [Detailed Revenue information in Hubspot](https://linear.app/impargo/issue/DFS-87/detailed-revenue-information-in-hubspot)
- [ ] [[Backend] Can't start a previously assigned order](https://linear.app/impargo/issue/DFS-85/[backend]-cant-start-a-previously-assigned-order)
- [ ] [[DriverApp] uploading Documents to finished orders](https://linear.app/impargo/issue/DFS-84/[driverapp]-uploading-documents-to-finished-orders)
- [ ] [[DriverApp] render flex overflows in some screens](https://linear.app/impargo/issue/DFS-83/[driverapp]-render-flex-overflows-in-some-screens)
- [ ] [[DriverApp] bloc exception is thrown when reconnecting to company](https://linear.app/impargo/issue/DFS-82/[driverapp]-bloc-exception-is-thrown-when-reconnecting-to-company)
- [ ] [Map reset when refreshing order list](https://linear.app/impargo/issue/DFS-81/map-reset-when-refreshing-order-list)
- [ ] [FirebaseAuthException: [firebase_auth/too-many-requests] We have blocked all requests from this device due to unusual ac...](https://linear.app/impargo/issue/DFS-80/firebaseauthexception-[firebase-authtoo-many-requests]-we-have-blocked)
- [ ] [Map view does not load in production app](https://linear.app/impargo/issue/DFS-79/map-view-does-not-load-in-production-app)
- [ ] [Fleet only showing first leg of route](https://linear.app/impargo/issue/DFS-78/fleet-only-showing-first-leg-of-route)
- [ ] [DriverApp stop based tracking flow with loading units](https://linear.app/impargo/issue/DFS-77/driverapp-stop-based-tracking-flow-with-loading-units)
- [ ] [DriverApp stop based tracking flow without loading units](https://linear.app/impargo/issue/DFS-76/driverapp-stop-based-tracking-flow-without-loading-units)
- [ ] [Automatically create notifications in hubspot when payment issues occur](https://linear.app/impargo/issue/DFS-75/automatically-create-notifications-in-hubspot-when-payment-issues)
- [ ] [Add stripe invoice id to BB invoices](https://linear.app/impargo/issue/DFS-74/add-stripe-invoice-id-to-bb-invoices)
- [ ] [Route calculation stats no longer accurate](https://linear.app/impargo/issue/DFS-73/route-calculation-stats-no-longer-accurate)
- [ ] [Implement drivers details view](https://linear.app/impargo/issue/DFS-72/implement-drivers-details-view)
- [ ] [Implement fleet map adaptions](https://linear.app/impargo/issue/DFS-71/implement-fleet-map-adaptions)
- [ ] [Implement updated fleet drivers list](https://linear.app/impargo/issue/DFS-70/implement-updated-fleet-drivers-list)
- [ ] [Back-end adaptions to needed to query list view of drivers efficiently](https://linear.app/impargo/issue/DFS-69/back-end-adaptions-to-needed-to-query-list-view-of-drivers-efficiently)
- [ ] [[DriverApp] Looking up a deactivated widget's ancestor is unsafe Exception](https://linear.app/impargo/issue/DFS-68/[driverapp]-looking-up-a-deactivated-widgets-ancestor-is-unsafe)
- [ ] [[DriverApp]the map native view breaks sometimes in a non consistent pattern](https://linear.app/impargo/issue/DFS-67/[driverapp]the-map-native-view-breaks-sometimes-in-a-non-consistent)
- [ ] [[DriverApp] Tracking is re-enabled each time the settings screen opens](https://linear.app/impargo/issue/DFS-66/[driverapp]-tracking-is-re-enabled-each-time-the-settings-screen-opens)
- [ ] [[Driverapp]: Show notification banner for DA messages.](https://linear.app/impargo/issue/DFS-65/[driverapp]-show-notification-banner-for-da-messages)
- [ ] [[CargoApps]: Ongoing order ETA is not shown in the fleet](https://linear.app/impargo/issue/DFS-64/[cargoapps]-ongoing-order-eta-is-not-shown-in-the-fleet)
- [ ] [Stop-based tracking KPI - DriverApp](https://linear.app/impargo/issue/DFS-63/stop-based-tracking-kpi-driverapp)
- [ ] [[Design-Implementation] Stop-based tracking Fleet](https://linear.app/impargo/issue/DFS-62/[design-implementation]-stop-based-tracking-fleet)
- [ ] [[CMS] Migrate landing page content into CMS](https://linear.app/impargo/issue/DFS-61/[cms]-migrate-landing-page-content-into-cms)
- [ ] [[CMS] Fix remaining issues and prepare migration](https://linear.app/impargo/issue/DFS-60/[cms]-fix-remaining-issues-and-prepare-migration)
- [ ] [Re-initializing map and location panning when switching screens](https://linear.app/impargo/issue/DFS-59/re-initializing-map-and-location-panning-when-switching-screens)
- [ ] [Update order details implementation](https://linear.app/impargo/issue/DFS-58/update-order-details-implementation)
- [ ] [[Shipperportal]: Invite carrier broken for existing carriers.](https://linear.app/impargo/issue/DFS-57/[shipperportal]-invite-carrier-broken-for-existing-carriers)
- [ ] [[ShipperPortal]: Choosing address when creating transport.](https://linear.app/impargo/issue/DFS-56/[shipperportal]-choosing-address-when-creating-transport)
- [ ] [Update drawer implementation for better UX](https://linear.app/impargo/issue/DFS-55/update-drawer-implementation-for-better-ux)
- [ ] [[DriverApp] Refactor order details bloc to use order id instead of constant order object](https://linear.app/impargo/issue/DFS-54/[driverapp]-refactor-order-details-bloc-to-use-order-id-instead-of)
- [ ] [[DriverApp] Newly activated DriverApp doesn't show chat messages unless user restarts the app](https://linear.app/impargo/issue/DFS-52/[driverapp]-newly-activated-driverapp-doesnt-show-chat-messages-unless)
- [ ] [[DriverApp] Country Flag resets when user request verification code](https://linear.app/impargo/issue/DFS-51/[driverapp]-country-flag-resets-when-user-request-verification-code)
- [ ] [Replace mailchimp email lists with hubspot](https://linear.app/impargo/issue/DFS-48/replace-mailchimp-email-lists-with-hubspot)
- [ ] [Identify visitors via email for hub-spot tracking](https://linear.app/impargo/issue/DFS-45/identify-visitors-via-email-for-hub-spot-tracking)
- [ ] [Enforce cookie policy using hubspot cookie banner + API](https://linear.app/impargo/issue/DFS-44/enforce-cookie-policy-using-hubspot-cookie-banner-api)
- [ ] [Order of assigned orders reversed in DriverApp](https://linear.app/impargo/issue/DFS-43/order-of-assigned-orders-reversed-in-driverapp)
- [ ] [[DriverApp] Tracking remains open after driver disconnects](https://linear.app/impargo/issue/DFS-42/[driverapp]-tracking-remains-open-after-driver-disconnects)
- [ ] [Disconnecting from company doesn't work](https://linear.app/impargo/issue/DFS-41/disconnecting-from-company-doesnt-work)
- [ ] [Onboarding deactivated App](https://linear.app/impargo/issue/DFS-40/onboarding-deactivated-app)
- [ ] [[DevOps] Configs and secrets handling](https://linear.app/impargo/issue/DFS-38/[devops]-configs-and-secrets-handling)

**Comment by david-paulus-saad** [comment]
Welcome to the release candidate. To see which issues are included in this release, check the done column in your team's respective currently active cycle:
- Planner & Orders squad: https://linear.app/impargo/team/POS/cycle/active
- DriverApp & Fleet squad: https://linear.app/impargo/team/DFS/cycle/active

**Comment by david-paulus-saad** [comment]
@jlabeit please, see the format of the two above comments, do you prefer the first or the second format, that would be attached to the release-candidate PR?

**Comment by KhaledSakr** [comment]
I also think the second comment is enough. The first comment will need more work if we want it to be useful (currently it prints all done issues, including issues from previous sprints, also non related issues from different repos). It would be a nice addition, but currently it's not worth the effort.

File: action.yml [hunk 1]
```diff
@@ -24,13 +21,32 @@ runs:
...
         git rebase origin/${{ inputs.target-branch }}
-        git push --force origin release-candidate 
+        git push --force-with-lease origin release-candidate 
+    - name: create release-candidate branch without rebasing
+      if: ${{ failure() }}
+      shell: bash
+      run: |
+        git config --global push.default simple
+        git checkout -b release-candidate
+        git push origin release-candidate
```
Comment (david-paulus-saad) [inline action.yml:49]: fallback to this step, when there is a conflict with the master branch.

**Review by KhaledSakr — APPROVED** [review]

**Pull request merged**